ui/ember-cli-build.js

```diff
@@ -94,6 +94,7 @@ module.exports = function (defaults) {
   app.import(
     'node_modules/@hashicorp/design-system-components/dist/styles/@hashicorp/design-system-components.css'
   );
+  app.import('node_modules/@hashicorp/vault-reporting/dist/styles/vault-reporting.css');
```

Comment:
Let's chat about this. I suspect there is no work-around, but having been the person who removed a lot of our third party css importing I'd like to understand why we can't use HDS css.

Reply:
Note from catchup: Look into other ways of handling the CSS

Reply:
I spent a bit of time investigating this and did not get to a good spot with lazy loading or completely isolating the styles but I have done a couple of things that I believe makes the style loading a bit more intentional:

1. Told the app scss plugin [to include the vault-reporting addon](https://github.com/hashicorp/vault/pull/30261/files#diff-0278030588e1fd822838c970d5a996997086d5e7256933fe145d965eb729b448R57) style path
2. Created a [new page stylesheet](https://github.com/hashicorp/vault/pull/30261/files#diff-023ec095d749299614417e31cfd052d9b5f8e571b4bc6c9737beb4ce2c4a5f17R5) that imports the vault-reporting css